iroh/src/commands/start.rs

```diff
@@ -256,12 +256,17 @@ async fn make_rpc_endpoint(
     iroh_data_root: &Path,
 ) -> Result<impl ServiceEndpoint<ProviderService>> {
     let rpc_addr = SocketAddrV4::new(Ipv4Addr::LOCALHOST, rpc_port);
-    let server_config = iroh::node::make_server_config(
+    let mut transport_config = quinn::TransportConfig::default();
+    transport_config
+        .max_concurrent_bidi_streams(MAX_RPC_STREAMS.try_into().unwrap())
```

Comment:
Again, quinn `VarInt` has `From<u8>`, `From<u16>`, and `From<u32>`.  Let's change the type of the constant to avoid the unwrap.

Reply:
side note, horrible api on quinns side..